dokka-runners/dokka-gradle-plugin/src/classicMain/kotlin/DokkaClassicPlugin.kt

```diff
@@ -13,7 +13,7 @@ import org.gradle.kotlin.dsl.withType
 import org.gradle.util.GradleVersion
 import org.jetbrains.dokka.DokkaDefaults
 
-open class DokkaPlugin : Plugin<Project> {
+open class DokkaClassicPlugin : Plugin<Project> {
```

Comment:
Renamed for clarity.

This rename should not affect user's project as the newly added code contains a class with the same FQN.

https://github.com/Kotlin/dokka/blob/26eba3f6b11f4d6c2a5c5e6e610557c3952cfc58/dokka-runners/dokka-gradle-plugin/src/main/kotlin/DokkaPlugin.kt#L21